tests/foreman/virtwho/test_ui_virtwho.py

```diff
@@ -626,3 +626,68 @@ def test_positive_virtwho_manager_role(session, test_name, form_data):
         # Delete the created user
         session.user.delete(username)
         assert not session.user.search(username)
+
+
+def test_positive_overview_label_name(form_data, session):
+    """Verify the label name on virt-who config Overview Page.
+
+    :id: 68422042-648c-4652-a52b-5bf42462d2ae
+
+    :CaseLevel: Integration
+
+    :CaseImportance: Medium
```

Comment:
please add bugzilla as part of test docstring ..

```
:BZ: 1649928
```

Reply:
ok, thanks